BLSQ/iaso
# IA-1677: add account name  + tooltip source version

The topbar only show the username but the account name should be also displayed for the user.
## Self proof reading checklist

## Changes
- add account name in Topbar
- create new component  `CurrentUserInfos` to display show source name, version number, commit hash 


## How to test

Go on any page and check in Topbar if user name and account name are well displayed and the `Popover` is working on hover displaying the source name, version number and hash commit if there is one

## Print screen / video

![Capture d’écran du 2022-12-05 16-31-35](https://user-images.githubusercontent.com/25134301/205677958-db29f7c4-e73d-41c0-ab82-01da76916bfe.png)

![Capture d’écran du 2022-12-05 16-31-44](https://user-images.githubusercontent.com/25134301/205677975-d418dedd-2a56-490d-af26-784b305694cc.png)

![Capture d’écran du 2022-12-05 16-32-03](https://user-images.githubusercontent.com/25134301/205677984-94c02fbb-67ed-4145-9c05-a1cbbb6de92a.png)
